concent_api/core/tests/test_unit_message_handlers.py

```diff
@@ -69,6 +77,30 @@ def test_that_messages_are_stored_with_correct_timestamps(self):
             parse_iso_date_to_timestamp(self.report_computed_task_timestamp)
         )
 
+    def test_that_update_subtask_passes_validations_with_next_deadline_as_int_type(self):
+        subtask = store_subtask(
+            task_id=self.task_to_compute.compute_task_def['task_id'],
+            subtask_id=self.task_to_compute.compute_task_def['subtask_id'],
+            provider_public_key=self.provider_public_key,
+            requestor_public_key=self.requestor_public_key,
+            state=Subtask.SubtaskState.REPORTED,
+            next_deadline=None,
+            task_to_compute=self.task_to_compute,
+            report_computed_task=self.report_computed_task,
+        )
+        subtask_state = Subtask.objects.get(subtask_id=self.task_to_compute.compute_task_def['subtask_id']).state
+        self.assertEqual(subtask_state, Subtask.SubtaskState.REPORTED.name)
+
+        update_subtask(
+            subtask=subtask,
+            state=Subtask.SubtaskState.FORCING_RESULT_TRANSFER,
+            next_deadline=int(self.task_to_compute.compute_task_def['deadline']) + settings.CONCENT_MESSAGING_TIME,
+            force_get_task_result=self.force_get_task_result,
+            set_next_deadline=True,
+        )
+        subtask_state = Subtask.objects.get(subtask_id=self.task_to_compute.compute_task_def['subtask_id']).state
+        self.assertEqual(subtask_state, Subtask.SubtaskState.FORCING_RESULT_TRANSFER.name)
+
```

Comment:
Please also make a test which will update subtask without next_deadline (passing to passive state)

Reply:
done